src/Raven.Server/Documents/Handlers/Admin/AdminLogsHandler.cs

```diff
@@ -42,7 +43,7 @@ public async Task SetConfiguration()
 
                 var configuration = JsonDeserializationServer.Parameters.SetLogsConfigurationParameters(json);
 
-                LoggingSource.Instance.SetupLogMode(configuration.Mode, Server.Configuration.Logs.Path.FullPath);
+                LoggingSource.Instance.SetupLogMode(configuration.Mode, Server.Configuration.Logs.Path.FullPath, configuration.LogRetention);
```

Comment:
if this will be null, then use the value from configuration

Reply:
done